includes/class-wc-install.php

```diff
@@ -307,8 +307,9 @@ private static function create_cron_jobs() {
 		wp_clear_scheduled_hook( 'woocommerce_tracker_send_event' );
 
 		$ve = get_option( 'gmt_offset' ) > 0 ? '-' : '+';
-
-		wp_schedule_event( strtotime( '00:00 tomorrow ' . $ve . get_option( 'gmt_offset' ) . ' HOURS' ), 'daily', 'woocommerce_scheduled_sales' );
+		$abs_offset = absint( get_option( 'gmt_offset' ) );
+		
+		wp_schedule_event( strtotime( '00:00 tomorrow ' . $ve . $abs_offset . ' HOURS' ), 'daily', 'woocommerce_scheduled_sales' );
```

Comment:
Should be better to just change to:

```php
wp_schedule_event( strtotime( '00:00 tomorrow ' . $ve . absint( get_option( 'gmt_offset' ) ) . ' HOURS' ), 'daily', 'woocommerce_scheduled_sales' );
```

No need for an extra variable here.

Reply:
Fair enough. That would work.

Reply:
Updating now.